### PR TITLE
Add NL quickstart sample. Fix some other quickstarts.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: bigquery/cloud-client/src/main/java/com/example/bigquery/QuickstartSample.java [hunk 1]
```diff
@@ -41,4 +41,3 @@ public static void main(String... args) throws Exception {
     System.out.printf("Dataset %s created.%n", dataset.getDatasetId().getDataset());
   }
 }
-// [END bigquery_quickstart]
```
Comment (jerjou) [inline bigquery/cloud-client/src/main/java/com/example/bigquery/QuickstartSample.java]: Er? What's with this? What about the start tag?

File: language/cloud-client/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Getting Started with Google Cloud Natural Language API and the Google Cloud Client libraries
+
+[Google Cloud Natural Language API][language] provides natural language
+understanding technologies to developers, including sentiment analysis, entity
+recognition, and syntax analysis. This API is part of the larger Cloud Machine
+Learning API.
```
Comment (jerjou) [inline language/cloud-client/README.md]: -> ...part of the larger collection of Cloud Machine Learning APIs

File: language/cloud-client/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+	mvn clean package -DskipTests
+
+You can then run a given `ClassName` via:
+
+	mvn exec:java -Dexec.mainClass=com.example.language.ClassName \
+	    -DpropertyName=propertyValue \
+		-Dexec.args="any arguments to the app"
+
+### Analyize a string for sentiment (using the quickstart sample)
```
Comment (jerjou) [inline language/cloud-client/README.md]: Analyze

File: language/cloud-client/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Build your project with:
+
+	mvn clean package -DskipTests
+
+You can then run a given `ClassName` via:
+
+	mvn exec:java -Dexec.mainClass=com.example.language.ClassName \
+	    -DpropertyName=propertyValue \
+		-Dexec.args="any arguments to the app"
```
Comment (jerjou) [inline language/cloud-client/README.md]: I'd do:

    -Dexec.args="arg1 'arg 2' arg3"

to make clear that spaces in a single arg have to be quoted.

File: language/cloud-client/src/main/java/com/example/language/QuickstartSample.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.example.language;
+
+// [START language_quickstart]
+// Imports the Google Cloud client library
+import com.google.cloud.language.spi.v1.LanguageServiceClient;
```
Comment (jerjou) [inline language/cloud-client/src/main/java/com/example/language/QuickstartSample.java]: Do you know why this is in a different package than the rest?

File: language/cloud-client/README.md (hunk 4 above)
Comment (jmdobry) [inline language/cloud-client/README.md]: Done

File: language/cloud-client/README.md (hunk 3 above)
Comment (jmdobry) [inline language/cloud-client/README.md]: Done

**Comment by jmdobry** [comment]
Addressed comments.

File: datastore/cloud-client/src/main/java/com/example/datastore/QuickstartSample.java [hunk 6]
```diff
@@ -46,4 +46,3 @@ public static void main(String... args) throws Exception {
     System.out.printf("Saved %s: %s%n", task.getKey().getName(), task.getString("description"));
   }
 }
-// [END datastore_quickstart]
```
Comment (jmdobry) [inline datastore/cloud-client/src/main/java/com/example/datastore/QuickstartSample.java]: All fixed. You are right, some of them were mistakes. Some of the deletions are legit though.

Comment (jerjou) [inline datastore/cloud-client/src/main/java/com/example/datastore/QuickstartSample.java]: Oh I see. Cool

**Comment by jerjou** [comment]
@lesv FYI

**Comment by jmdobry** [comment]
Can be merged?

**Pull request merged**